tests/ui/specs/Panel/Agate-Panel-Page.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
```

Comment:
you don't need to use '-' for Page file name.
Agate-PanelPage.js  is Better.
spec and view is mandatory for ui-test-utils. but Page is not

Reply:
Ok, thank you!